test/Makefile

```diff
@@ -1,11 +1,14 @@
 beams := $(patsubst %.erl, %.beam, $(wildcard *.erl))
 
 .PHONY: compile
-compile: $(beams)
+compile: $(beams) otp_version
 
 %.beam: %.erl
 	erlc +debug_info -o $@ $<
 
+otp_version:
+	erl +V 2>&1 | awk '{ print $$NF }' | awk -F. '{ print $$1 }' | xargs expr 11 + | tr -d '\n' > otp_version
```

Comment:
I made it possible to save Erlang/OTP version into `otp_version` file on build time of beam files.
The `otp_version` file is used in version-dependent tests (e.g., `test_try_stacktrace`).